packages/syft/src/syft/service/queue/queue.py

```diff
@@ -218,6 +224,25 @@ def handle_message_multiprocessing(
         status = Status.COMPLETED
         job_status = JobStatus.COMPLETED
     except Exception as e:
+        root_context = AuthedServiceContext(
+            server=context.server,
+            credentials=worker.signing_key.verify_key,  # type: ignore
+        )
+        link = LinkedObject.with_context(
+            queue_item, context=root_context, service_type=QueueService
+        )
+        message = CreateNotification(
+            subject=f"Job {queue_item.job_id} failed!",
+            from_user_verify_key=worker.signing_key.verify_key,  # type: ignore
+            to_user_verify_key=credentials,
+            linked_obj=link,
+            notifier_types=[NOTIFIERS.EMAIL],
+            email_template=FailedJobTemplate,
+        )
+
+        method = worker.get_service_method(NotificationService.send)
```

Comment:
maybe switch out with worker.services.notification.send based on the slack thread from before?

Reply:
Fixed by: [9dc189d](https://github.com/OpenMined/PySyft/pull/9280/commits/9dc189d66e6fa84db2a4db1b7f2449a3d20e8ff6)